CMakeLists.txt

```diff
@@ -90,10 +90,10 @@ endif()
 
 if (
     (NOT TARGET mimalloc) AND 
-    (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/mimalloc/CMakeLists.txt) AND
-    (${ARCHITECTURE} STREQUAL "x86_64")
+    (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/mimalloc/CMakeLists.txt)
     )
-        set(MI_BUILD_TESTS OFF)
+        set(MI_BUILD_SHARED OFF CACHE BOOL "dont build a dll" FORCE) #not building a dll allows building on 32 bit, otherwise there's some errors on init.c about not finding a imported symbol
```

Comment:
>dont build a dll [](start = 44, length = 16)

Nitpick: I guess the description pertains to MI_BUILD_SHARED, so it should sound like "build a dll?"
Same applies for the below flag about building the tests.

Reply:
Used the exact same description as the one from mimalloc

---
In reply to: [459659420](https://github.com/Azure/azure-c-pal/pull/14#discussion_r459659420) [](ancestors = 459659420)